docs/usage/exporting_models.md

```diff
@@ -494,13 +468,15 @@ t = Transaction(
 
 print(t.model_dump())
 #> {'id': '1234567890'}
-# TODO this is wrong! not all of "user" should be excluded
+# TODO: this is wrong! not all of "user" should be excluded
+# TODO: do we need to fix the type of the argument to Field? Or do we want to just remove that functionality?
```

Comment:
The line `    user: User = Field(exclude={'username'})` above is actually a type error for the `exclude` argument. It seems we changed how that argument works. I noticed this before, but wasn't quite sure how we want this to work. @samuelcolvin @adriangb thoughts?

This section may need a bigger rework if we aren't going to fix that.